stellarphot/settings/models.py

```diff
@@ -558,10 +622,42 @@ class PhotometryOptionalSettings(BaseModelWithTableRep):
     False
     """
 
-    include_dig_noise: bool = True
-    reject_too_close: bool = True
-    reject_background_outliers: bool = True
-    fwhm_by_fit: bool = True
+    include_dig_noise: Annotated[
+        bool,
+        Field(
+            description=(
+                "Should the digitization noise be included in the calculation of "
+                "the noise for each observation?"
+            )
+        ),
+    ] = True
+
+    reject_too_close: Annotated[
+        bool,
+        Field(
+            description="Should sources that are too close be excluded from photometry?"
+        ),
+    ] = True
+
+    reject_background_outliers: Annotated[
+        bool,
+        Field(
+            description=(
+                "Should extreme pixels (e.g. from a star in the annulus) "
```

Comment:
For readability, I would move the parenthetical to the end of the sentence and before the question mark.